.drone.yml

```diff
@@ -183,6 +183,91 @@ volumes:
   host:
     path: /var/run/docker.sock
 
+---
+kind: pipeline
+name: default-s390x
+
+node:
+  arch: s390x
+
+platform:
+  os: linux
+  arch: amd64
```

Comment:
I believe this is on purpose, because Drone does not support s390x arch platform.

Reply:
Indeed, that is a work around that we have in order to be able to run the pipelines: https://confluence.suse.com/display/EN/Documentation#Documentation-Specialneedsfors390x